docs/source/user_guide/01_philosophy.rst

```diff
@@ -1,55 +1,7 @@
 
-The philosophy and the approach
-===============================
+The philosophy
```

Comment:
Not silly at all. I love that it's tied to the inspiration. I've always loved those spatial cube rotations too. @jhamrick Have you ever met @geekygirldawn? I think you would find each other's research interesting.